x/mint/keeper/keeper_test.go

```diff
@@ -0,0 +1,104 @@
+package keeper_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/suite"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+
+	sdksimapp "github.com/cosmos/cosmos-sdk/simapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/tendermint/tendermint/crypto"
+
+	"github.com/certikfoundation/shentu/v2/simapp"
+)
+
+// shared setup
+type KeeperTestSuite struct {
+	suite.Suite
+	app *simapp.SimApp
+	ctx sdk.Context
+}
+
+func (suite *KeeperTestSuite) SetupTest() {
+	suite.app = simapp.Setup(false)
+	suite.ctx = suite.app.BaseApp.NewContext(false, tmproto.Header{Time: time.Now().UTC()})
+	coins := sdk.Coins{sdk.NewInt64Coin("uctk", 80000*1e6)}
+	suite.Require().NoError(sdksimapp.FundModuleAccount(suite.app.BankKeeper, suite.ctx, "mint", coins))
+}
+
+func (suite *KeeperTestSuite) TestKeeper_SendToCommunityPool() {
+	tests := []struct {
+		name  string
+		coins sdk.Coins
+		err   bool
+	}{
+		{
+			name:  "Funding Community Pool",
+			coins: sdk.NewCoins(sdk.NewInt64Coin(suite.app.StakingKeeper.BondDenom(suite.ctx), 100*1e6)),
+			err:   false,
+		},
+		{
+			name:  "Funding With 0",
+			coins: sdk.NewCoins(sdk.NewInt64Coin(suite.app.StakingKeeper.BondDenom(suite.ctx), 0)),
+			err:   true,
+		},
+	}
+
+	for _, tc := range tests {
+		suite.T().Log(tc.name)
+		moduleAcct := sdk.AccAddress(crypto.AddressHash([]byte("mint")))
+		if tc.err {
+			err := suite.app.MintKeeper.SendToCommunityPool(suite.ctx, tc.coins)
+			suite.Require().Nil(err)
```

Comment:
Since we are expecting an error, shouldn't it be:
```suggestion
			suite.Require().Error(err)
```
I see that the function `SendToCommunityPool` returns `nil` for "0" fund value. Should this be changed to a custom error?

Reply:
No since it's mostly executed on an endblocker